app/models/invitation.rb

```diff
@@ -6,18 +6,21 @@ class Invitation < ActiveRecord::Base
   belongs_to :invitee, class_name: "User", inverse_of: :invitations
   belongs_to :actor, class_name: "User", inverse_of: :invitations
 
-  after_commit :notify_invitation_invited, on: :create
-
   aasm column: :state do
-    state(:invited, {
-      initial: true,
-      before_enter: [:generate_code, :associate_existing_user]
-    })
+    state :pending, initial: true
+    state :invited do
+      validates :invitee, presence: true
+    end
     state :accepted
     state :rejected
 
+    event(:invite, {
+      after: [:generate_code, :associate_existing_user],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.